app/routes/games/list.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions: {
+    newGame: function () {
+      // The user must not be anonymous.
+      if (this.get('session.user.isAnonymous')) {
```

Comment:
Note: at the moment it'll look like you don't have any decks because they are not connected with your user record.